src/content/whats-new/2021/10/instant-observability-10-13-21.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Get Instant Observability with New Relic I/O'
+summary: 'Leverage 400+ open source quickstarts to get started monitoring your stack'
+releaseDate: '2021-10-13'
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/instant-observability-quickstarts'
+getStartedLink: 'https://onenr.io/06vjAmnD3jP'
+---
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=c9zprczTXj8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
I couldn't get this this doc to build in localhost, but I can see it in Gatsby build. Except, both videos are gray boxes. Can you see the videos? If yes I'll approve this PR. If no, you may want to try using the format I sent in slack for a video in a different doc. That does use the "title" which yours does not. Maybe that's required for YouTube?

Reply:
@barb, I can see these two videos now that I've corrected the embed links.